### PR TITLE
 task-LtFDCXvu

### PR DESCRIPTION
**Pull request by BYavorskyj**
task-LtFDCXvu added css-property and showed date/time 
https://trello.com/c/LtFDCXvu/46-main-show-date-time
@AntonShalenui @vkryviak-ua

### CODE REVIEW

File: src/client/components/Main.js [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
+			.then(user => this.setState({ username: user.username }));
 	}
 
 	render() {
-		const {username, showTextAfterImage} = this.state;
+		const { username, showTextAfterImage } = this.state;
 		return (
 			<main className="global-main">
 				{username ? <h1>{`Hello ${username}`}</h1> : <h1>Loading.. please wait!</h1>}
+				{/*<div className="dateAndTime"> {`${dateAndTime}`} </div>*/}
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: Do not leave old code. For this we have a GIT

File: src/client/components/main.css [hunk 2]
```diff
@@ -4,3 +4,8 @@
 .main-img {
     width:200px;
 }
+.dateAndTime {
+	position: absolute;
```
Comment (vkryviak-ua) [inline src/client/components/main.css]: Will be better to use `float:right`

File: src/client/components/Main.js [hunk 3]
```diff
@@ -1,29 +1,37 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import ReactImage from '../react.png';
 import './main.css';
 import Article from './Article';
 
+let dateAndTime = new Date();
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: All of this variables are `const`

File: src/client/components/Main.js [hunk 4]
```diff
@@ -1,29 +1,37 @@
...
+let year = dateAndTime.getFullYear();
+let mount = dateAndTime.getMonth()+1;
+let day = dateAndTime.getDate();
+let hour = dateAndTime.getHours();
+let minutes = dateAndTime.getMinutes();
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: Probably will be nice to have here an enter

File: src/client/components/main.css (hunk 2 above)
Comment (BYavorskyj) [reply src/client/components/main.css]: But then the left picture is mixed.

File: src/client/components/Main.js (hunk 1 above)
Comment (BYavorskyj) [reply src/client/components/Main.js]: ok

File: src/client/components/Main.js (hunk 3 above)
Comment (BYavorskyj) [reply src/client/components/Main.js]: fixed

File: src/client/components/main.css (hunk 2 above)
Comment (BYavorskyj) [reply src/client/components/main.css]: used float:right and padding:10px

**Pull request merged**